styleguide.html

```diff
@@ -63,4 +62,19 @@
     <input type="checkbox" id="styleguide" />
     <label for="styleguide">Checkbox</label>
   </p>
-</form>
\ No newline at end of file
+</form>
+
+<form>
+  <textarea></textarea>
+</form>
+
+<div class="select-container">
+  <label for="field_language">Languages</label>
+  <select name="language" id="field_language" required="">
```

Comment:
unrequired quotes all over, and simply writing `required` (no `=""`) is enough too.